src/Components/Push.tsx

```diff
@@ -0,0 +1,129 @@
+import { Session } from '@supabase/supabase-js';
+import Constants from 'expo-constants';
+import * as Device from 'expo-device';
+import * as Notifications from 'expo-notifications';
+import { useState, useEffect, useRef } from 'react';
+import { View, Button, Platform } from 'react-native';
+
+import supabase from '../supabase/createClient';
+
+Notifications.setNotificationHandler({
+  handleNotification: async () => ({
+    shouldShowAlert: true,
+    shouldPlaySound: false,
+    shouldSetBadge: false,
+  }),
+});
+
+async function getDummyUpdate() {
```

Comment:
Can we keep notifications config in this file, but eventually move actual notification functions (including example notifications) to a separate `notifications.ts` file in the supabase queries folder. Or somewhere else, idk what necessarily works best (?)